src/58MP.yaml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Flow mapping edge cases
+  from: '@ingydotnet'
+  tags: edge flow mapping
+  yaml: |
+    {"x"
```

Comment:
this is practically the same as https://github.com/yaml/yaml-test-suite/blob/main/src/4MUZ.yaml
So maybe add it to the existing one.

Reply:
Done in 7ba5479